trusted-types/default-policy-callback-arguments.html

```diff
@@ -47,7 +47,7 @@
       [ "about:blank", "TrustedScriptURL", "HTMLScriptElement src",
           _ => script.src = "about:blank" ],
       [ "2+2", "TrustedScript", "eval", _ => eval("2+2") ],
-      [ "(function anonymous(\n) {\nreturn 2+2\n})", "TrustedScript",
+      [ "function anonymous(\n) {\nreturn 2+2\n}", "TrustedScript",
```

Comment:
Perhaps worth mentioning explicitly in the commit message, especially to help when this will be imported back to chromium (IIUC, this was based on Chromium's behavior but this is not aligned with https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-function-constructor).